objectRecognitionSource/README.md

```diff
@@ -222,6 +228,13 @@ is set to be either "local" or "both".
 captured). If not specified, the value will default to 1 which saves every captured image.
 *   labelImage: Optional. If set to "true", images will be saved with bounding boxes and labels. If set to "false", or if not 
 set at all, the images will be saved with no bounding boxes or labels.
+*   savedResolution: Optional. A map containing options for adjusting the resolution of the saved images.
+    * *Options for savedResolution:*
+    1. longEdge: Optional. Used to resize the saved images. Must be a non-negative integer that is smaller than the long edge 
```

Comment:
I think I'd change _Used to resize saved _images._ to _This sets the maximum long edge dimension for saved images._. By using this phraseology, we establish our _contract_ which then means that we don't have to worry about what happens if we save things that are smaller (because they were smaller), etc.